packages/bindiff.vm/bindiff.vm.nuspec

```
@@ -2,13 +2,14 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>bindiff.vm</id>
-    <version>8.0.0.20250219</version>
+    <version>8.0.0.20250505</version>
     <authors>Zynamics</authors>
-    <description>A comparison tool for binary files that assists in quickly finding differences and similarities in disassembled code</description>
+    <description>Bindiff is a comparison tool for binary files that finds differences and similarities in disassembled code.</description>
```

This tool name is also not the one used in the tool GH page:
```suggestion
    <description>BinDiff is a comparison tool for binary files that finds differences and similarities in disassembled code.</description>
```